lnd/lnd_wallet.go

```diff
@@ -256,11 +256,11 @@ func NewLndFeeEstimator(ctx context.Context, walletkit walletrpc.WalletKitClient
 	return &LndFeeEstimator{ctx: ctx, walletkit: walletkit}
 }
 
-func (l *LndFeeEstimator) GetFeePerKw(targetBlocks uint32) (float64, error) {
+func (l *LndFeeEstimator) GetSatsPerVByte(targetBlocks uint32) (float64, error) {
 	res, err := l.walletkit.EstimateFee(l.ctx, &walletrpc.EstimateFeeRequest{ConfTarget: int32(targetBlocks)})
 	if err != nil {
 		return 0, err
 	}
 
-	return float64(res.SatPerKw / 4000), nil
+	return float64(res.SatPerKw / 250), nil
```

Comment:
I think this should be 253 and not 250
https://github.com/lightninglabs/pool/blob/master/rpcserver.go#L684

Reply:
Or maybe it should stay 250, so its always slightly higher than 1 sat/vbyte

Reply:
This is just the inverse of the scale factor so `1000 / 4` https://github.com/lightningnetwork/lnd/blob/262591c3331c2170697fe89b8c49ab857db0adf7/lnwallet/chainfee/rates.go#L32

I would use `min(1.1, satsPerVByte)` afterwards to set the floor slightly higher if that is desired.

Also it might be better to use weight units throughout instead of sats/vbyte anyway - it seems to me that kwu is the canonical measure and the vbyte measure is an alternative that serves mostly to keep the bitcoincore fee rpc calls consistent

Reply:
Sorry I just responded to this now. I went looking for this file in LND just a bit ago. 250 sounds right to me.